content/shmem_bundle_stop.tex

```diff
@@ -0,0 +1,40 @@
+\apisummary{
+    Stop a communication bundle.
+}
+
+\begin{apidefinition}
+
+\begin{Csynopsis}
+void @\FuncDecl{shmem\_bundle\_stop}@(void);
+void @\FuncDecl{shmem\_ctx\_bundle\_stop}@(shmem_ctx_t ctx);
+\end{Csynopsis}
+
+\begin{apiarguments}
+    \apiargument{IN}{ctx}{A context handle specifying the context on which to
+    perform the optimization. When this argument is not provided, the
+    optimization is performed on the default context.}
+\end{apiarguments}
+
+\apidescription{
+    The \FUNC{shmem\_bundle\_stop} routine provides a hint to the \openshmem
```

Comment:
We are not defining the exit condition for this routine correctly - I'm not sure whether the intention is to have an implicit quiet as part of this routine. 
Based on the example - it looks like there is an implicit quiet. Not sure - why this is defined in such a way.

Reply:
As discussed in the WG's, there is no implicit quiet at the end of a bundle (but there is within the `shmem_free`, right?  I don't think the example needs a quiet, but maybe the example could be improved in this regard).

My intent was for the statement:
> [bundling routines] do not affect the completion or ordering semantics of any OpenSHMEM
 routines in the program

to cover this.  But perhaps it would help to include something like this in the API description or a note?